fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/expression/ExpressionNode.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query.expression;
+
+
+/**
+ *
+ */
+public interface ExpressionNode {
+
+}
```

Comment:
not used?

Reply:
Removed, along with unused child classes